app/models/log_file.rb

```diff
@@ -267,4 +267,5 @@ def self._request_logs(options)
     _log.info("#{log_header} #{msg}")
     task.update_status("Queued", "Ok", msg)
   end
+  private_class_method :_request_logs
```

Comment:
I think this is meant to be public and needs to be moved before the private keyword.  See above, we put something on the queue, to be run:

```ruby
       MiqQueue.put(
         :class_name   => name,
         :method_name  => "_request_logs",
         :args         => [options],
         :zone         => zone,
         :miq_callback => cb,
         :msg_timeout  => LOG_REQUEST_TIMEOUT,
         :priority     => MiqQueue::HIGH_PRIORITY
       )
```

So, this is a public method with a bad name.